js/components/twittax.js

```diff
@@ -11,6 +11,7 @@ class Twax extends LitElement {
   constructor() {
     super();
     this.user = {};
+    this.image = '';
```

Comment:
```suggestion
    this.image = '/assets/default_profile_400x400.png';
```
Pour avoir une valeur par défaut